File: reunet-api/src/main/java/com/reunet/app/models/payload/LoginResponse.java
package com.reunet.app.models.payload;

import com.reunet.app.models.User;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@ToString
public class LoginResponse {
    private User user;
    private String token;

    public LoginResponse(User user, String token) {
        this.user = user;
        this.token = token;
    }
}

File: front_movil/app/src/main/java/com/reunet/app/models/payload/response/LoginResponse.kt
package com.reunet.app.models.payload.response

import com.google.gson.annotations.SerializedName
import com.reunet.app.models.User
import java.io.Serializable

data class LoginResponse(
    @SerializedName("user") var user: User,
    @SerializedName("token") var token: String
)
